src/types/TestObject.test.ts

```diff
@@ -0,0 +1,212 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as Test from './TestObjects'
+
+describe('TestObject', () => {
+
+    const ALPHA = "alpha"
+    const BETA = "beta"
+    const CHARLIE = "charlie"
+    const conversationId = "1234" 
+
+    const makePair = (sourceA: string, sourceB: string, result: Test.RatingResult): Test.RatingPair => {
+        const sourceNames = [sourceA, sourceB].sort()
+        return {
+            conversationId,
+            sourceNames: [sourceNames[0], sourceNames[1]],
```

Comment:
What is the purpose if this line?
It seems equivalent to `sourceNames`

Reply:
sourceNames needs to be alphabetically sorted

Reply:
Also it is typed as an array of length 2 and result from sort is not

Reply:
> sourceNames needs to be alphabetically sorted. 
They are sorted by `sort()` fuction not this line that takes them out of array and then puts them back in.

> it is typed as an array of length 2
Then specify this constraint on the `sourceNames` type instead of doing weird manipulation of values to simulate intention.
`const sourceNames = ['a', 'b'].sort() as [string, string]`